src/index.css

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
Add `scss` to the project and use it for styles. And also unnecessary styles should be removed.

Reply:
done